src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts

```diff
@@ -276,14 +280,18 @@ class VoiceChatSessions {
 				case SpeechToTextStatus.Recognizing:
 					if (text) {
 						session.controller.updateInput([inputValue, text].join(' '));
-						acceptTranscriptionScheduler.cancel();
+						if (delay > 0) {
+							acceptTranscriptionScheduler.schedule();
```

Comment:
Shouldnt this `cancel`?

Reply:
Yes that is correct, I will add the cancellation in the if statement and remove the schedule. I have test the auto stop feature on my machine and the first time if nothing is spoken co pilot keeps listening.  So I don't think we need to cancel it if delay == 0 (because it wasn't scheduled in the first place).